apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -6115,7 +6106,7 @@ typeahead-container .dropdown-menu li.active > a > a:hover {
     background-color: $alice-blue-light;
 }
 
-.ledg-head table td.pd-r25 {
+.ledg-head table td.pdR25imp {
```

Comment:
This change is reverted while according to the code design, we've fixed most of the camel case classes. @prateek-giddh

Reply:
Done